### PR TITLE
install tools on container

### PR DESCRIPTION
**Pull request by AFriemann**
* kubectl
* curl, openssl, gzip, tar, bash, ca-certificates

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,13 @@ FROM python:3-alpine
 
 LABEL maintainer="CloudOps <cloudops@clark.de>"
 
+# Install kubectl
+RUN apk add --no-cache openssl curl tar gzip bash ca-certificates
```
Comment (dcernag) [inline Dockerfile:6]: we could add these to the other RUN to avoid extra layers

File: Dockerfile [hunk 2]
```diff
@@ -2,6 +2,13 @@ FROM python:3-alpine
...
+RUN curl -L -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.8.0/bin/linux/amd64/kubectl && \
```
Comment (sandjaie) [inline Dockerfile:7]: Should we have the version in a var

File: Dockerfile (hunk 1 above)
Comment (dcernag) [reply Dockerfile:6]:
```
RUN \
  apk add --no-cache --upgrade git openssl curl tar gzip bash ca-certificates && \
  curl -L -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.8.0/bin/linux/amd64/kubectl && \
  chmod +x /usr/bin/kubectl && \
  kubectl version --client
  python3 /app/setup.py install && \
  apk del git && \
  rm -rf /app /var/cache/apk
```

Comment (AFriemann) [reply Dockerfile]: done

File: Dockerfile (hunk 2 above)
Comment (sandjaie) [reply Dockerfile]: 1.18.0 is latest though

Comment (AFriemann) [reply Dockerfile]: done :+1:

Comment (dcernag) [reply Dockerfile]: good idea, 1.8.0 looks old, was it supposed to be 1.18 ?

File: Dockerfile [hunk 3]
```diff
@@ -2,6 +2,16 @@ FROM python:3-alpine
 
 LABEL maintainer="CloudOps <cloudops@clark.de>"
 
+ARG KUBECTL_VERSION="1.8.0"
```
Comment (dcernag) [inline Dockerfile:5]: `1.18.0` ?
```suggestion
ARG KUBECTL_VERSION="1.18.0"
```

Comment (AFriemann) [reply Dockerfile]: yup

**Pull request merged**